openlibrary/macros/DonateModal.html

```diff
@@ -0,0 +1,95 @@
+$def with(book, size="M", ids={})
+
+$ title = book.title
+
+<style type="text/css">
```

Comment:
Not great that these styles are here, but I would be more ok with it if all the styles had a shared prefix; e.g. `donateModal__instructions` to avoid causing conflicts with other things.